app/src/main/java/fr/free/nrw/commons/AboutActivity.kt

```diff
@@ -142,7 +142,7 @@ class AboutActivity : BaseActivity() {
     }
 
     fun launchPrivacyPolicy(view: View?) {
-        Utils.handleWebUrl(this, Uri.parse(BuildConfig.PRIVACY_POLICY_URL))
+        Utils.handleWebUrl(this, Uri.parse("https://commons-app.github.io/#privacy-policy"))
```

Comment:
Why not modify `PRIVACY_POLICY_URL`? :-)

Reply:
`buildConfigField "String", "PRIVACY_POLICY_URL", "\"https://github.com/commons-app/commons-app-documentation/blob/master/android/Privacy-policy.md\""` in app/build.gradle (two places in that file, same value)

Reply:
By the way, would you mind using the value https://commons-app.github.io/privacy-policy as the URL? _(I moved the privacy policy to its own file after all, to increase our chances of passing Google Play inspection by being clearer as a separate page)_
Thanks! :-)